src/app/shared/components/template/template.module.ts

```diff
@@ -26,14 +28,16 @@ import { DEMO_COMPONENTS } from "packages/components/demo";
   imports: [
     CommonModule,
     FormsModule,
-    ReactiveFormsModule,
     IonicModule,
-    SharedPipesModule,
-    NouisliderModule,
     LottieModule,
+    MatSortModule,
+    MatTableModule,
```

Comment:
nit(non-blocking): I can't remember how well we've implemented optimisation for this module, but might be more viable to include the imports in the table component itself and turn into a standalone component so that the modules are easier to remove from build when the component is not needed